src/components/site/PageContent.md

```diff
@@ -0,0 +1,7 @@
+```jsx
+<p className="dg_content-summary">
+  The Collection Division is responsible for the collection of residential
+  trash, single stream recyclables and yard materials in Baltimore County. This
+  division oversees the County's 39 private trash and recycling collectors.
+</p>
+```
```

Comment:
This file does not seem to show up in the component library with `npm run`. The link on the main page does not take me to #section-page-content as well.

Reply:
@theilman7  this is working for me, https://dotgov-components-qynn2wiqp.now.sh/#section-page-content, does it work for you?

I'm also assuming you mean when you run locally with `npm start` it's not working?

(update) i suspect maybe because this was out of date. Think it may be working now.

Reply:
Checking again, will follow up shortly.

Reply:
The link you sent above displays the component, but attempting to view with `npm-start` still does not.

Reply:
I can view this now.